deploy/deploy_ai_service_registry.ts

```diff
@@ -0,0 +1,25 @@
+import {HardhatRuntimeEnvironment} from "hardhat/types"
+import {DeployFunction} from "hardhat-deploy/types"
+
+import {Controller} from "../typechain"
+
+import ContractDeployer from "../utils/deployer"
+
+const func: DeployFunction = async function(hre: HardhatRuntimeEnvironment) {
```

Comment:
I suggest following the structure of [this deploy script](https://github.com/livepeer/protocol/blob/delta/deploy/deploy_ticket_broker.ts) since in practice this deploy script would just deploy a new instance of the ServiceRegistry contract without deploying a new Controller.

Something like:

```
const deployResult = await deploy("ServiceRegistry", {
    from: deployer,
    args: [controllerDeployment.address],
    log: true
})
await deployments.save("AIServiceRegistry", deployResult)
```

- The first line would deploy a *non-proxy* version of the ServiceRegistry contract.
- The second line would save the metadata for the deployment under the name "AIServiceRegistry" in the `deployments` directory

Reply:
@yondonfu, understood. I've already deployed a single ServiceRegistry on [sepolia](https://sepolia.arbiscan.io/) so deploying one on Abritrum should cause no problems.

I now kept the AIServiceRegistry contract closer to the mainnet implementation. However, I agree that doing so might unnecessarily complicate matters. We can always consider migrating later if we decide to merge the subnet into the mainnet 👍🏻.

Reply:
@yondonfu done -> https://arbiscan.io/address/0x04c0b249740175999e5bf5c9ac1da92431ef34c5#writeContract.